homeassistant/components/livisi/livisi_entity.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         config_entry: ConfigEntry,
         coordinator: LivisiDataUpdateCoordinator,
         device: dict[str, Any],
+        use_room_as_device_name=False,
```

Comment:
I suggest to make this a keyword only argument, which will give more flexibility:
```suggestion
        *,
        use_room_as_device_name=False,
```